tooling-server-config/src/main/java/org/jboss/eap/qe/ts/common/server/config/BaseManagementClientFactory.java

```diff
@@ -0,0 +1,47 @@
+package org.jboss.eap.qe.ts.common.server.config;
+
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+import java.io.IOException;
+
+/**
+ * Provides common basic implementation of a factory for {@link ManagementClient} objects.
+ */
+public abstract class BaseManagementClientFactory implements ManagementClientFactory {
```

Comment:
This class seems to me like an unnecessary code. Wouldn't be just the interface sufficient?